tarodb/assets_store.go

```diff
@@ -1265,6 +1266,10 @@ func (a *AssetStore) LogPendingParcel(ctx context.Context,
 		txnID, err := q.UpsertChainTx(ctx, ChainTx{
 			Txid:  newAnchorTXID[:],
 			RawTx: anchorTxBytes,
+			HeightHint: sql.NullInt32{
```

Comment:
Re my prior comment, in this case, we'd actually store it below in `InsertAssetTransfer`.

Reply:
Se above comment. I also made `HeightHint` not nullable (since it is not added in a migration anymore).